package.json

```diff
@@ -128,9 +128,9 @@
     "docs": "./scripts/docs.sh",
     "lint": "jshint lib/ system-test/ test/ && jscs lib/ system-test/ test/",
     "test": "npm run docs && mocha test/*/*.js test/index.js test/docs.js",
-    "system-test": "mocha system-test/* --timeout 60s",
-    "cover": "istanbul cover -x 'system-test/*' _mocha -- --timeout 60s test/*/*.js test/docs.js system-test/*",
-    "coveralls": "istanbul cover -x 'system-test/*' _mocha --report lcovonly -- --timeout 60s test/*/*.js test/docs.js system-test/* -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
+    "system-test": "mocha system-test/* --timeout 60s --bail",
+    "cover": "istanbul cover -x 'system-test/*' _mocha -- --timeout 60s --bail test/*/*.js test/docs.js system-test/*",
```

Comment:
Do we want to bail for coverage reports? Even if a test fails in coverage, I believe you're still able to determine whether or not a certain piece of code is covered.